building/build-acis/flannel-monitor/build.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+set -e -u
+cd "$(dirname "$0")"
+source ../common/container-build-helpers.sh
+
+VERSION="0.1.5"
+
+BUILDVER="stretch.20180111T215606Z"
+UPDATE_TIMESTAMP="2018-01-17T17:12:00-0500"
+
+common_setup
+
+init_builder
+build_with_go
+
+GODIR="${BUILDDIR}/godir"
+mkdir "${GODIR}"
+cp -R "src" -t "${GODIR}"
+
+tar -C "${BUILDDIR}" -xf "${UPSTREAM}/kubernetes-src-v1.8.0.tar.xz" ./staging/src/k8s.io/ ./vendor/github.com/ ./vendor/golang.org/ ./vendor/gopkg.in/ ./vendor/k8s.io/kube-openapi/pkg/common
+# tar -C "${BUILDDIR}" -xf "${UPSTREAM}/prometheus-2.0.0.tar.xz" prometheus-2.0.0/vendor
+# tar -C "${BUILDDIR}" -xf "${UPSTREAM}/prometheus-client_golang-0.9.0-pre1.tar.xz" client_golang-0.9.0-pre1/
```

Comment:
What is the significance of the commented-out code?

Reply:
That should be removed.